lib/prepare-system.sh

```diff
@@ -5,6 +5,15 @@ source lib/functions.sh
 
 log "INFO" "Preparing your system for bridgehead installation ..."
 
+# Check, if running in WSL
+if [[ $(grep -i Microsoft /proc/version) ]]; then
+    # Check, if systemd is available
+    if [ $(systemctl is-system-running) -eq "offline" ]; then
```

Comment:
Tested it in the commandline using bash. Seems like Bash doesn't like -eq with strings.

Reply:
"" missing on the left side.